src/ripple/app/main/Application.cpp

```diff
@@ -1714,17 +1689,33 @@ ApplicationImp::setup()
 
     if (config_->reporting())
     {
-        reportingETL_->run();
+        reportingETL_->start();
     }
 
     return true;
 }
 
 void
-ApplicationImp::doStart(bool withTimers)
+ApplicationImp::start(bool withTimers)
 {
-    startTimers_ = withTimers;
-    start();
+    JLOG(m_journal.info()) << "Application starting. Version is "
+                           << BuildInfo::getVersionString();
+
+    using namespace std::chrono_literals;
+    if (withTimers)
+    {
+        setSweepTimer();
+        setEntropyTimer();
+    }
+
+    m_io_latency_sampler.start();
+    m_resolver->start();
+    m_loadManager->start();
+    m_shaMapStore->start();
+    overlay_->start();
```

Comment:
`overlay_` should not be started when running in reporting mode.

Reply:
`overlay_` is actually never even instantiated and is just `nullptr` when running in reporting mode.